### ISSUE TITLE
Outliner - Add Delete Object operator to hotkeys

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#1377]
# Issue
![bforartists_2020-02-18_21-50-42](https://user-images.githubusercontent.com/35357409/74797274-beee0d00-5298-11ea-869f-eda5e4ece7bd.png)
Right now, you can press "Delete" to delete a collection.. but you cannot, for some reason, delete objects or selections inside collection. 

But this operator to delete objects/selections already exists. 

# Solution
So, with the fans on Discord, we discovered that we can have both on the same hotkey with Press and Release on the "Delete" hotkey. 

So, we can use Press to delete an object or selection, and Release to delete a Collection. Now whatever is selected, object or collection - will delete correctly with the single Delete button, as expected. 

# Task
Add these changes in the image to the hotkey map. 

**TLDR**
Feature request: delete everything selected with press/release fusion to "delete" hotkey.

**Comment by ReinerBforartists** [comment]
How is this solved in Blender?

**Comment by Draise14** [comment]
It isn't applied in Blender keymaps. only the industry compatible keymap has both entries and then it's with two separate hotkeys.

I'll double check to make sure, someone else told me this one.

**Comment by Draise14** [comment]
Double checked, in the Blender keymap it just doesn't exist.

**Comment by ReinerBforartists** [comment]
Thanks for checking :)

**Comment by ReinerBforartists** [comment]
Fixed in master

**Comment by ReinerBforartists** [comment]
Doesn't work anymore.

**Comment by Draise14** [comment]
Tested with my BFA 2 - 2.0.0 (Config 2.83) map with the Dev Build 04-25-2020, setup still works.

**Comment by ReinerBforartists** [comment]
They must have changed something under the hood.  Both items have been back to Press. But that was not even the problem. I have to make delete collection with press, and delete object on release. The order has become important here.

**Comment by ReinerBforartists** [comment]
Fixed in master
